experiments/clef-hipe-2022/postprocess.py

```diff
@@ -0,0 +1,46 @@
+import string
+
+import click
+
+
+@click.command()
+@click.argument("predictions", type=click.File("r"))
+@click.argument("processed_predictions", type=click.File("w"))
+def postprocess(predictions, processed_predictions):
+    # We assume that names of these entity types don't start with a punctuation
+    # mark and remove B-<type_to_correct> annotations from punctuation marks
+    types_to_correct = ["pers", "loc", "date", "work"]
+    corrected_type = ""
+    for line in predictions:
+        line = line.strip()
+        # Headers and comments
+        if line.startswith("TOKEN") or line.startswith("#"):
+            processed_predictions.write(line + "\n")
+            corrected_type = ""
+            continue
+        # Empty lines
+        if not line:
+            processed_predictions.write("\n")
+            corrected_type = ""
+            continue
+        elems = [e.strip() for e in line.split("\t")]
```

Comment:
Nitpick: Nachdem ich dem Code entnehme dass `elems` immer zwei Werte hat könntest du den Code etwas lesbarer machen indem du hier in zwei Variable destructurst:

```suggestion
        token, tag = [e.strip() for e in line.split("\t")]
```

Dann wären die beiden Blöcke unten leichter zu verstehen weil man nicht immer vom Array-Index auf die Semantik mappen muss.

Reply:
Danke für den Hinweis! Tatsächlich hat jede Zeile 10 Einträge. Für das Postprocessing interessieren uns nur die ersten beiden, aber wir wollen alle Einträge später wieder ausgeben. 
Ich glaube, es ist fast verwirrender, alle 10 Einträge namentlich aufzuzählen, oder?

Reply:
Ah, definitiv, sorry, das war dann zu viel reininterpretiert von mir 🤗